cdap-ui/app/cdap/components/DataPrep/TopPanel/SchemaModal.js

```diff
@@ -155,7 +155,10 @@ export default class SchemaModal extends Component {
     let workspaceId = DataPrepStore.getState().dataprep.workspaceId;
     let filename = `${workspaceId}-schema.json`;
 
-    let data = JSON.stringify(this.state.schema, null, 4);
+    let fields = this.state.schema;
+
+    // TODO: Change this when we make UI schema consistent with backend schema (JIRA: CDAP-13010)
+    let data = JSON.stringify([getSchemaObjWrapper(getSchemaObj(fields))], null, 4);
```

Comment:
If you are making a utility function, it's much better to just call a single function that gives you what you need. So perhaps add a third function that will combine the `getSchemaObjWrapper` and `getSchemaObj`